docs/tutorials/buttons.rst

```diff
@@ -111,41 +118,43 @@ This is called *nesting*. We use nesting to define blocks of code like this::
 
     display.show(Image.SURPRISED)
 
-The ``running_time`` function returns the number of milliseconds since the
-device started.
+Funkcja ``running_time`` zwraca liczbę milisekund od startu urządzenia.
 
-The ``while running_time() < 10000:`` line checks if the running time is less
-than 10000 milliseconds (i.e. 10 seconds). If it is, *and this is where we can
-see scoping in action*, then it'll display ``Image.ASLEEP``. Notice how this is
-indented underneath the ``while`` statement *just like in our to-do list*.
+Linia ``while running_time() < 10 000:`` sprawdza czy czas pracy urządzenia
+jest mniejszy od 10 000 milisekund (czyli 10 sekund). Jeżeli tak, *i to jest
+miejsce gdzie możemy zobaczyć skalę działania*, to zostanie wyświetlony
+``Image.ASLEEP``. Zauważ jak to jest wcięte poniżej instrukcji ``while``
+*tak jak na naszej liście zadań*.
 
-Obviously, if the running time is equal to or greater than 10000 milliseconds
-then the display will show ``Image.SURPRISED``. Why? Because the ``while``
-condition will be False (``running_time`` is no longer ``< 10000``). In that
-case the loop is finished and the program will continue after the ``while``
-loop's block of code. It'll look like your device is asleep for 10
-seconds before waking up with a surprised look on its face.
+Oczywiście, jeśli czas pracy jest równy lub większy niż 10 000 milisekund,
+wówczas na ekranie pojawi się ``Image.SURPRISED``. Dlaczego? Ponieważ warunek
+``while`` (ang. dopóki) będzie fałszywy (ang. False) (``running_time`` nie jest już ``< 10000``).
+W takim przypadku pętla jest zakończona i program będzie kontynuowany po bloku
+kodu pętli ``while``. Wygląda na to, że twoje urządzenie śpi przez 10 sekund
+zanim obudzi się z zaskoczoną miną na swojej twarzy.
 
-Try it!
+Wypróbuj to!
 
-Handling an Event
+Ogsługa zdarzenia
 +++++++++++++++++
 
-If we want MicroPython to react to button press events we should put it into
-an infinite loop and check if the button ``is_pressed``.
+Jeśli chcemy aby MicroPython reagował na zdarzenia naciśnięcia przycisku, to
+powinniśmy zdarzenie to umieścić w nieskończonej pętli i sprawdzać czy przycisk
```

Comment:
"to powinniśmy" → "musimy"

Reply:
Nie zdarzenie mamy umieścić, bo to bez sensu, tylko microbita. Może coś takiego:

", musimy wprowadzić go w nieskończoną pętlę i sprawdzać w niej czy przycisk jest wciśnięty (ang. ``is_pressed``).